vumi/transports/dmark/dmark_ussd.py

```diff
@@ -19,6 +19,10 @@ class DmarkUssdTransportConfig(HttpRpcTransport.CONFIG_CLASS):
         " expires.",
         default=600, static=True)
 
+    fix_to_addr = ConfigBool(
+        "Whether or not to ensure that the to_addr is always starting with a "
+        "* and ending with a #", default=False, static=True)
```

Comment:
Static configs are only required for things that are used in transport setup or whatever. Dynamic configs are looked up dynamically in places where dynamic overrides are available (such as vumi-go channel definitions) but fall back to a static lookup otherwise. The per-message overhead is negligible.